core/peerset/src/lib.rs

```diff
@@ -78,10 +153,10 @@ impl From<u64> for IncomingIndex {
 #[derive(Debug)]
 pub struct PeersetConfig {
 	/// Maximum number of ingoing links to peers.
-	pub in_peers: u32,
+	pub in_peers: usize,
```

Comment:
Minor nit: could stay a `u32`. It's not representing anything related to memory.